File: output-plot/boxplot.R
input <- paste("~/Box Sync/JUNO/R programs/WDM-paper/OFC/rawData-storageAR-K10-N4-W2000-load20-num100GIP2-runIndex0.csv", sep = "")
data <- read.csv(input, header=TRUE, sep=",")
data <- as.matrix(data)
data <- data[,1]/1000

boxplot(data, outline=FALSE)


input <- paste("~/Box Sync/JUNO/R programs/WDM-paper/OFC/rawData-storageAR-K10-N4-W2000-load20-num100GIP4-runIndex0.csv", sep = "")
data1 <- read.csv(input, header=TRUE, sep=",")
data1 <- as.matrix(data1)
data1 <- data1[,1]/1000

boxplot(data1, outline=FALSE)

File: src/boxplot.py
import sys, csv
import pickle

n = int(sys.argv[1])
output = "rawData-storageAR-K10-N4-W2000-load20-lossRate1e-11-num100GIP" + str(n) + "-runIndex0.csv"
with open(output, 'rb') as pfile:
	storageUsage = pickle.load(pfile)

plt.boxplot(storageUsage)
plt.show()